kimchi/src/circuits/polynomials/keccak/circuitgates.rs

```diff
@@ -189,6 +189,64 @@ where
     }
 }
 
+//~ ##### `KeccakBits` - 32-bit decomposition gate
+//~
+//~ * This circuit gate is used to constrain that two values of 64 bits are decomposed
+//~   correctly in two halves of 32 bits. It will be used to constrain all inputs and
+//~   intermediate values of the XOR gates.
+//~ * This gate operates on the `Curr` row.
+//~ * This is not a definitive gate. It may be integrated with other gates in the future.
+//~
+//~ It uses one type of constraints
+//~ * copy    - copy to another cell (32-bits to the XOR gate, and 64-bits to the RangeCheck gate)
+//~
+//~ | Column |      `Curr`   |
+//~ | ------ | ------------- |
```

Comment:
Yeah I know, but for now I just wanted to have one gate doing it so I could build more realistic examples. And then when I have the full Keccak gadget I will figure out where to move these rows, because I am convinced they will be mergeable somewhere else